app/scripts/router.js

```diff
@@ -103,6 +105,7 @@ function (
       'settings/avatar/crop(/)': showView(AvatarCropView),
       'settings/avatar/gravatar(/)': showView(AvatarGravatarView),
       'settings/avatar/camera(/)': showView(AvatarCameraView),
+      'settings/avatar/gravatar_permissions(/)': showView(GravatarPermissions),
```

Comment:
Can you add this new screen and its events to the [client-metrics doc](https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md)?